test/is.test.js

```diff
@@ -108,24 +108,44 @@ describe('`is`', function () {
             checkThat: 'https://youtube.come/qwerty',
             is: false
         },
-        'checks valid steam id': {
+        'checks valid steam profile name': {
+            fn: 'steam.name',
+            checkThat: 'abcdefgh123',
+            is: true
+        },
+        'checks invalid steam profile name': {
+            fn: 'steam.name',
+            checkThat: ' ',
+            is: false
+        },
+        'checks valid steam profile id': {
             fn: 'steam.id',
-            checkThat: '12345678901234567',
+            checkThat: '1234567890123456',
             is: true
         },
-        'checks invalid steam id': {
+        'checks invalid steam profile id': {
             fn: 'steam.id',
-            checkThat: 'qwert.y',
+            checkThat: 'x',
+            is: false
+        },
+        'checks valid steam profile id': {
+            fn: 'steam.profileUrl',
+            checkThat: 'https://steamcommunity.com/profiles/1234567890123456',
```

Comment:
Sorry I didn't see this earlier, but isn't this URL a bit short? A real Steam profile seems to have a 17 character string and fails the validation, but this is 16 chars.